### DISCUSSION TITLE
CORS Error

### DISCUSSION THREAD

**Discussion by eric-sison** [logto-io/logto#5405] [Q&A]
I'm getting a CORS error, despite the fact that I have added my app's origin domain, `http://localhost:3000`, in the Admin app.

My app configuraion:
![Screenshot from 2024-02-13 15-13-53](https://github.com/logto-io/logto/assets/65406753/c820330a-4ed8-4aa5-8323-0276a62d18dd)

The error I get in the browser:
![Screenshot from 2024-02-13 15-14-36](https://github.com/logto-io/logto/assets/65406753/d9b90de5-deca-4ded-ba58-7ad35abd98b7)

Here, I'm trying to call the change password api endpoint.

**Comment by eric-sison** [comment]
Update: Same thing, when using the Logto cloud service.

My app configuration:
![Screenshot from 2024-02-14 14-36-15](https://github.com/logto-io/logto/assets/65406753/4d786dbf-aa5b-40be-aeda-ba57816a8678)

Browser error:
![Screenshot from 2024-02-14 14-34-52](https://github.com/logto-io/logto/assets/65406753/b87d85bb-f1d2-40bd-b3a7-5b3dc26c55eb)

**Comment by wangsijie** [comment]
CORS is supported only for OIDC, specifically for URLs starting with "/oidc". CORS configurations for the management API are not available yet.

**Comment by wangsijie** [comment]
For security reasons, it's advised to avoid making management API requests from the frontend. Instead, perform these requests on a backend server.
